server/storage-access-layer/src/main/java/ai/starwhale/mlops/storage/fs/StorageAccessServiceFile.java

```diff
@@ -134,4 +146,9 @@ public void delete(String path) throws IOException {
             }
         }
     }
+
+    @Override
+    public String signedUrl(String path, Long expTimeMillis) throws IOException {
+        return serviceProvider + "/" + path + "/" + (System.currentTimeMillis() + expTimeMillis);
```

Comment:
Do we need to encrypt this info?

Reply:
no need for now i think

Reply:
User can get others data when controller uses local fs.

Reply:
if user use local fs in production he will suffer a lot of other issues besides security